src/components/play/layer-settings/LayerSettings.js

```diff
@@ -72,6 +73,10 @@ const styles = theme => ({
         backgroundColor: '#333333',
         [theme.breakpoints.down('md')]: {
             height: 48,
+            marginBottom: 10,
+        },
+        [theme.breakpoints.down('sm')]: {
```

Comment:
Isn't this in effect @media queries? Could you use this for isMobile?

I actually have no clue tbh

Reply:
you're 100% right, I think it was a temp hack that turned into a component wide soln, on it ASAP.